controllers/shooter.py

```diff
@@ -0,0 +1,51 @@
+import math
+
+from wpimath.geometry import Translation2d
+
+from magicbot import StateMachine, state, timed_state, default_state
+
+from components.chassis import ChassisComponent
+from components.intake import IntakeComponent
+from components.shooter import ShooterComponent
+from utilities.game import get_goal_speaker_position
+
+
+class Shooter(StateMachine):
+
+    shooter_component: ShooterComponent
+    chassis: ChassisComponent
+    intake: IntakeComponent
+
+    def translation_to_goal(self) -> Translation2d:
+        return (
+            get_goal_speaker_position().toTranslation2d()
+            - self.chassis.get_pose().translation()
+        )
+
+    @default_state
+    def idling(self):
+        self.shooter_component.set_range(self.translation_to_goal().norm())
+
+    @state(first=True)
+    def aiming(self):
+        self.shooter_component.set_range(self.translation_to_goal().norm())
```

Comment:
What is the point of setting the range in line 31 when it’s going to be updated again in line 41